src/Driver/Driver.jl

```diff
@@ -366,6 +368,8 @@ Recognized keyword arguments are:
         run for the specified time (in simulation seconds)
 - `fixed_number_of_steps::Int = -1`:
         if `≥0` perform specified number of steps
+- `degree::NTuple{2, Int} = -1,-1`:
+        tuple of horizontal and vertical polynomial degrees for spatial discretization order (no space before/after comma)
```

Comment:
```suggestion
        tuple of horizontal and vertical polynomial degrees for spatial discretization order
```
As this is Julia code, a tuple and its syntax are well-defined.

Reply:
Sorry! From our last conversation I understood that we wanted to make sure to document the no comma requirement (especially for `bash`), but yeah here it was not needed. I went with auto-pilot mode, adding it to too many places!